api/controllers/projects.go

```diff
@@ -264,6 +265,8 @@ func PostProjectMembers(
 
 	if can {
 		var seats int64
+		usersInMemberRoles, _ := Repo.UsersInMemberRoles(input.Members)
+
```

Comment:
I'm not sure calling the function here, then again in ProjectAddMembers is a good choice, but passing the result of UsersInMemberRoles as argument to ProjectAddMembers would be a mess too.